content/blog/2020/2020-4-015-covid-19-part-3.md

```diff
@@ -0,0 +1,112 @@
++++
+author = "Andreja Kovačič"
+date = "2020-04-13"
+draft = false
+title = "Data Mining COVID-19 Epidemics: Part 3"
+type = "blog"
+thumbImage = "/blog_img/2020/2020-04-09-choropleth.png"
+frontPageImage = "/blog_img/2020/2020-04-09-choropleth.png"
+categories = ["covid-19", "visualization", "addons", "trends", "time"]
+shortExcerpt = "Inspection Covid-19 data trends"
+longExcerpt = "Inspecting and comparing Covid-19 data, absolute and relative growth."
++++
+
+# Data Mininig COVID-19 Epidemics: Part 3
+
+So far, we've seen how to do [basic visualisations](https://orange.biolab.si/blog/2020/2020-04-02-covid-19-basic/) related to the corona virus and how to look at the [developments on the map](https://orange.biolab.si/blog/2020/2020-04-09-covid-19-part-2/). Be sure to check that out first, before delving into this one. 
```

Comment:
developments on the map --> disease progression on a map?

Reply:
that out first -> them out first?

Reply:
"do visualisations" ali "make visualisations"?